--- a/build.gradle
+++ b/build.gradle
@@ -115,7 +115,7 @@
 
 dependencies {
     compile group: 'org.codehaus.groovy', name: 'groovy-all', version: project.hasProperty('gpars_groovyVersion') ? gpars_groovyVersion : '2.1.9'
-    compile 'org.codehaus.jsr166-mirror:jsr166y:1.7.0'
+    compile 'org.codehaus.jsr166-mirror:extra166y:1.7.0'
     compile('org.multiverse:multiverse-core:0.7.0') { transitive = false }
     compile group: 'org.jboss.netty', name: 'netty', version: project.hasProperty('gpars_nettyVersion') ? gpars_nettyVersion : '3.2.9.Final'
     compile 'org.codehaus.jcsp:jcsp:1.1-rc5'
@@ -323,7 +323,7 @@
 task zipDist(type: Zip) {
     from jar.outputs.files
     from(runtimeClasspath) {
-        include('jsr166*', 'netty*', 'multiverse*')
+        include('extra166*', 'netty*', 'multiverse*')
     }
     from('licenses') {
         include '*'
